docs/zkapps/o1js/custom-tokens.mdx

```diff
@@ -0,0 +1,211 @@
+---
```

Comment:
Make sure it 100% corresponds to what we currently have for https://docs.minaprotocol.com/
(including content itself + correct belonging to the docs structure)
Custom tokens is no published for the users I think.

Reply:
Not displayed - see sidebar.js